amp_conf/htdocs/admin/page.modules.php

```diff
@@ -129,6 +129,20 @@
 		$online = 0;
 		unset($modules_online);
 	} else {
+		// check for local-only modules and see if they have online update data
+		foreach(array_diff(array_keys($modules_local), array_keys($modules_online)) as $name) {
+			if ($name === "builtin") continue;
+			if (!empty($modules_local[$name]['updateurl']) && parse_url($modules_local[$name]['updateurl'], PHP_URL_SCHEME) === 'https') {
+				$module_update_json = file_get_contents($modules_local[$name]['updateurl']);
```

Comment:
No proxy support. Will crash interface if file doesn't exist (server is down etc). Using FreePBX::Curl() is probably a better idea.